python-package/xgboost/spark/core.py

```diff
@@ -0,0 +1,880 @@
+# type: ignore
+"""Xgboost pyspark integration submodule for core code."""
+# pylint: disable=import-error, consider-using-f-string, too-many-arguments, too-many-locals
+# pylint: disable=invalid-name, fixme, too-many-lines, unbalanced-tuple-unpacking, no-else-return
+# pylint: disable=protected-access, logging-fstring-interpolation, no-name-in-module
+# pylint: disable=wrong-import-order, ungrouped-imports, too-few-public-methods, broad-except
+# pylint: disable=too-many-statements
```

Comment:
Please fix as many errors as possible and skip only if it conflicts with designs that cannot be fixed in near future. Some are easy like invalid-name and consider-using-f-string . Linter might be annoying but it's necessary.